nncf/onnx/quantization/quantize_model.py

```diff
@@ -234,6 +239,11 @@ def compress_weights_impl(
     if model.opset_import[0].version < 21 and group_size > 0:
         msg = "ONNX models with opset version < 21 do not support block-wise quantization."
         raise nncf.ValidationError(msg)
+
+    external_data_dir = get_external_data_dir(advanced_parameters)
+    if external_data_dir:
+        model.metadata_props.add(key="nncf.external_data_dir", value=external_data_dir)
```

Comment:
Will NNCF return a model with metadata_props `nncf.external_data_dir`?

Reply:
This is a good question. I will check. We probably shouldn't do that.

Adding this parameter to `metadata_props` is just one possible solution (the simplest one that requires minimal changes). We could also pass such a parameter explicitly.

Reply:
Fixed